src/features/classification/applied-by-ai-classification-reason/__tests__/AppliedByAiClassificationReason.test.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { render, screen } from '../../../../test-utils/testing-library';
+import AppliedByAiClassificationReason from '../AppliedByAiClassificationReason';
+import messages from '../messages';
+
+describe('AppliedByAiClassificationReason', () => {
+    const defaultProps = {
```

Comment:
Prefer to create objects dynamically in `beforeEach` so that they can be modified without affecting subsequent tests